coalib/processes/Processing.py

```diff
@@ -338,7 +341,30 @@ def instantiate_processes(section,
         log_printer,
         ignored_file_paths=glob_list(section.get('ignore', "")),
         limit_file_paths=glob_list(section.get('limit_files', "")))
+
+    # This stores all matched files irrespective of whether coala is run
+    # only on changed files or not. Global bears require all the files
+    complete_filename_list = filename_list
+
+    # We need to compute the changed files so that the cache gets updated
+    # even if caching is disabled (in which case all files will be processed
+    # but the cache will be updated).
+    changed_files = cache.get_changed_files(
+        filename_list) if cache else filename_list
+
+    # If caching is enabled then the local bears should process only the
+    # changed files.
+    if section.get('caching', False):
+        # FIXME: Log this to the debug channel instead.
+        log_printer.info("Caching is enabled, results from previous "
+                         "runs may not appear.")
```

Comment:
the message is wrong isn't it? Results should all appear unless we have a bug but *log messages* may not be there